### DISCUSSION TITLE
verse_onprem/3.2_test/user/faq_run_standalone

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#81] [Q&A]
# verse_onprem/3.2_test/user/faq_run_standalone

Beginning with HCL Verse 2.0, you can install and run HCL Verse as a standalone browser app that is based on progressive web app (PWA) technology.

https://help.hcl-software.com/verse_onprem/3.2_test/user/faq_run_standalone.html

**Comment by garima-shringi** [comment]
Test for notification
